Utilities/bin_patch.h

```diff
@@ -34,9 +34,12 @@ enum class patch_type
 	lef64,
 	be16,
 	be32,
+	bd32,
```

Comment:
what do these 3 new types stand for ?

Reply:
2 types are hints that they don't contain executable code, I tried to explain in commit message.

Reply:
then maybe add a comment in code for both